zipkin-server/receiver-zipkin-http/src/main/java/zipkin/server/receiver/zipkin/http/ZipkinHTTPReceiverConfig.java

```diff
@@ -17,58 +17,5 @@
 import org.apache.skywalking.oap.server.library.module.ModuleConfig;
 
 public class ZipkinHTTPReceiverConfig extends ModuleConfig {
-  private String restHost;
-  private int restPort;
-  private String restContextPath;
-  private int restMaxThreads = 200;
-  private long restIdleTimeOut = 30000;
```

Comment:
What does this removing mean? Could you explain the diff between `receiver-zipkin-http` and `http-query-plugin`. They seem to be consufed.

Reply:
First, in the v2 version of Zipkin, they share the same HTTP port when receiver and query. So I moved the HTTP configuration in the HTTP Receiver and Querier into the core. 
The receiver is used to collect spans from agents, and the Query is used to query the span for UI.

Reply:
I think that is how sharing server works. The receiver is still able to work on the separate port, especially query port is exposed for the internet, I assume a separate port is better.
Could you check the OAP sharing server and take a reference to optimize the receiver?

Reply:
I have separated the export ports for the receiver and querier modules (query span and health). Now, they can utilize the default HTTP server in the core module if the port is defined as `-1`. Otherwise, the module will start a new HTTP server and use it.